packages/itwinui-react/src/core/List/ListItem.tsx

```diff
@@ -33,6 +33,7 @@ const ListItemComponent = React.forwardRef((props, ref) => {
       data-iui-size={size === 'large' ? 'large' : undefined}
       data-iui-actionable={actionable ? 'true' : undefined}
       data-iui-focused={focused ? 'true' : undefined}
+      tabIndex={disabled ? undefined : -1}
```

Comment:
undo changes in this file. `tabIndex` should be managed by the consumer of this component.

Reply:
`tabIndex` has been moved from `ListItem` to `TransferList.ListItem`, but I think it's still necessary to define it for the keyboard navigation. Correct me if I am wrong on that

Reply:
yes, `TransferList.ListItem` is the "consumer" of this component, so that's the right place to set tabindex

Reply:
I think we will want to use `-1` for all items though, including disabled items.

Disabled items need to be focusable and should have `aria-disabled`, but clicking or pressing <kbd>Enter</kbd> while focusing them should do nothing.

Reply:
This has been fixed